projects/CMRxRecon/tools/create_data_dir.py

```diff
@@ -0,0 +1,177 @@
+import argparse
+import logging
+import pathlib
+from argparse import RawTextHelpFormatter
+
+from create_data_with_masks import ACCELERATIONS, create_data_with_masks
+from create_symlinks import create_symlinks
+
+logger = logging.getLogger("CreateTrainingData")
+logger.setLevel(logging.INFO)
+
+# Create a file handler and set the format
+log_file = "./CreateTrainingData.log"
+file_handler = logging.FileHandler(log_file)
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+file_handler.setFormatter(formatter)
+
+# Add the file handler to the logger
+logger.addHandler(file_handler)
+
+# Define the available options for the 'data_type' argument
+DATA_TYPES = ["Cine", "Mapping"]
+
+ASSUMED_BASE_PATH_STRUCTURE = """
+    base_path
+    ├── MultiCoil
+    │   ├── Cine_or_Mapping
+    │   │   ├── TrainingSet
+    │   │   │   ├── FullSample
+    │   │   │   ├── AccFactor04
+    │   │   │   ├── AccFactor08
+    │   │   │   └── AccFactor10
+    │   │   ├── ValidationSet
+    │   │   │   ├── AccFactor04
+    │   │   │   ├── AccFactor08
+    │   │   │   └── AccFactor10
+    │   │   ├── TestSet
+    │   │   │   ├── AccFactor04
+    │   │   │   ├── AccFactor08
+    │   │   │   └── AccFactor10
+    """
+
+SYMLINKS_PATH_STRUCTURE = """
+    target_path
+    ├── MultiCoil
+    │   ├── training
+    │   │   ├── P001_T1map.h5
+    │   │   ├── with_masks_P001_T1map.h5
+    │   │   ├── P001_cine_sax.h5
+    │   │   ├── with_masks_P001_cine_sax.h5
+    │   ├── Cine_or_Mapping
+    │   │   ├── validation
+    │   │   │   ├── AccFactor04
+    │   │   │   |   ├── P001_<..>.h5
+    │   │   │   ├── AccFactor08
+    │   │   │   |   ├── P001_<..>.h5
+    │   │   │   └── AccFactor10
+    │   │   │   |   ├── P001_<..>.h5
+    │   │   ├── test
+    │   │   │   ├── AccFactor04
+    │   │   │   |   ├── P001_<..>.h5
+    │   │   │   ├── AccFactor08
+    │   │   │   |   ├── P001_<..>.h5
+    │   │   │   └── AccFactor10
+    │   │   │   |   ├── P001_<..>.h5
+    """
+
+# Create an argument parser
+parser = argparse.ArgumentParser(
+    description="Process data and create symlinks",
+    formatter_class=RawTextHelpFormatter,
+)
+
+# Add arguments for base path, target path, and data type
```

Comment:
Add all of this to a `def main()` and then call it in an `if __name__ == ...`

Reply:
Done